apps/src/templates/instructions/RubricField.jsx

```diff
@@ -98,7 +121,11 @@ class RubricField extends Component {
               disabledMode={this.props.disabledMode}
             />
           )}
-          <details style={styles.detailsArea}>
+          <details
+            id={`rubric-details-${this.props.rubricLevel}`}
+            style={styles.detailsArea}
+            open={this.props.expandByDefault}
```

Comment:
this is also not safe for multiple renderings. instead, you need to initialize state.detailsOpen from props.expandByDefault

Reply:
am I understanding correctly that `detailsOpen` should be controlling whether this `<details` element is expanded? if so, you would also need to set `open={this.state.detailsOpen}` here